snapcraft/internal/lifecycle.py

```diff
@@ -60,7 +60,7 @@
   my-part:
     # See 'snapcraft plugins'
     plugin: nil
-"""  # noqa, lines too long.
+""" # noqa, lines too long.
```

Comment:
It's usually a good idea to check the `git diff` before making the commit, to make sure that only the lines you intended changed. In this case it's just a whitespace so no big deal.